demos/spatio-temporal/input_data.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""
+Updated on  Wed Mar 11 09:28:50 2020
+
+@author: 
+"""
+
+import numpy as np
+import pandas as pd
+import pickle as pkl
+
+def load_sz_data(dataset):
+    sz_adj = pd.read_csv(r'data/sz_adj.csv',header=None)
+    adj = np.mat(sz_adj)
+    sz_tf = pd.read_csv(r'data/sz_speed.csv')
+    return sz_tf, adj
+
+def load_los_data(dataset):
+    los_adj = pd.read_csv(r'data/los_adj.csv',header=None)
```

Comment:
What's the `los` data? It would be best if we could add it to `stellargraph/datasets/datasets.py` instead of having an extra script for the demo. If it ends up being too awkward to add it there, feel free to open an issue for follow up work.

Reply:
Yes, it can be easily added to the `stellargraph/datasets/datasets.py`. 
 There are a number of other methods that I think of as data related such as formatting the data in a time series to be fed into a time series forecasting model. So having it all in one utility file where anyone interested in just doing spatio-temporal stuff can look them, instead of looking them up in a centralized  utility file.  This was just for convenience. I can merge it in the `stellargraph/datasets/datasets.py`.